### PR TITLE
termusic: add pages

### PR DESCRIPTION
**Pull request by GianniBYoung**
- [X] The page(s) are in the correct platform directories: `linux`.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** v0.7.2

### CODE REVIEW

**Comment by kbdharun** [comment]
This command should be placed under `pages/linux` directory. And not in pages directly.

File: pages/linux/termusic.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# termusic
+
+> A terminal music player written in Rust that uses vim-like key bindings.
+> More information: <https://github.com/tramhao/termusic>.
+
+- Open termusic to a specific directory. **This can be set permanantly in `~/.config/termusic/config.toml`.**:
+
+`termusic <PATH/TO/MUSIC>`
+
+- View termusic's usage info:
```
Comment (kbdharun) [inline pages/linux/termusic.md:10]:
```suggestion
- Disable the album cover:

`termusic -c {path/to/music_file}`
- View termusic's usage info:
```
I think adding this flag would be useful too 
image

**Review by kbdharun — CHANGES_REQUESTED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks @GianniBYoung

File: pages/linux/termusic.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# termusic
+
+> A terminal music player written in Rust that uses vim-like key bindings.
+> More information: <https://github.com/tramhao/termusic>.
+
+- Open termusic to a specific directory. (It can be set permanently in `~/.config/termusic/config.toml`):
+
+`termusic {path/to/music}`
```
Comment (kbdharun) [inline pages/linux/termusic.md:8]:
```suggestion
`termusic {path/to/directory}`
```
I think this would be better

File: pages/linux/termusic.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Disable the album cover:
```
Comment (marchersimon) [inline pages/linux/termusic.md:10]:
```suggestion
- Disable showing the album cover for a specific file:
```

**Review by marchersimon — APPROVED** [review]
Looks good. Thanks @GianniBYoung

**Review by kbdharun — APPROVED** [review]
LGTM

**Pull request merged**